### PR TITLE
Enable interactive commands

### PR DESCRIPTION
**Pull request by drognanar**
Enables reset interactive from project/copy to interactive/execute in interactive commands.

### CODE REVIEW

**Comment by tannergooding** [comment]
@dotnet-bot, test prtest/lin/tst/unit32 please

**Comment by drognanar** [comment]
FYI @dotnet/roslyn-interactive

**Comment by davkean** [comment]
Ignore the prtest/lin/tst/unit32 failure, we're testing out new jobs.

@tannergooding Please make note of the failure.

**Comment by amcasey** [comment]
LGTM, as long as https://github.com/dotnet/roslyn/pull/8053 is merged as well (i.e. needs tests).

**Comment by kuhlenh** [comment]
I'm going to push back on adding 2 new commands to the Context Menu (we're trying not to clutter the menu). Can we follow the F# model here and just have "Send To Interactive"? The functionality here should follow what you've done for CopyToInteractiveWindow.

**Comment by drognanar** [comment]
test this please

**Comment by drognanar** [comment]
test prtest/win/dbg/eta please

**Comment by drognanar** [comment]
retest vsi please

**Comment by kuhlenh** [comment]
:+1:

File: src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct [hunk 1]
```diff
@@ -296,10 +295,8 @@
     <KeyBinding guid="guidInteractiveWindowCmdSet" id="cmdidSearchHistoryPrevious" key1="VK_UP" mod1="Control Alt" editor="guidInteractiveWindow" />
     <KeyBinding guid="guidInteractiveWindowCmdSet" id="cmdidCopyCode" key1="C" mod1="Control Shift" editor="guidInteractiveWindow" />
 
-    <KeyBinding guid="guidInteractiveWindowCmdSet" id="cmdidExecuteInInteractiveWindow" editor="GUID_TextEditorFactory" mod1="Control" key1="E" mod2="Control" key2="VK_RETURN"  />
+    <KeyBinding guid="guidInteractiveWindowCmdSet" id="cmdidExecuteInInteractiveWindow" editor="guidCSharpEditorFactory" mod1="Control" key1="E" mod2="Control" key2="E" />
```
Comment (davkean) [inline src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct]: Is there a reason we aren't using the existing core commands?

Comment (drognanar) [inline src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct]: I made the change to reuse `execute in interactive` core command

**Comment by drognanar** [comment]
test prtest/determinism please

File: src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct [hunk 2]
```diff
@@ -296,10 +285,8 @@
     <KeyBinding guid="guidInteractiveWindowCmdSet" id="cmdidSearchHistoryPrevious" key1="VK_UP" mod1="Control Alt" editor="guidInteractiveWindow" />
     <KeyBinding guid="guidInteractiveWindowCmdSet" id="cmdidCopyCode" key1="C" mod1="Control Shift" editor="guidInteractiveWindow" />
 
-    <KeyBinding guid="guidInteractiveWindowCmdSet" id="cmdidExecuteInInteractiveWindow" editor="GUID_TextEditorFactory" mod1="Control" key1="E" mod2="Control" key2="VK_RETURN"  />
+    <KeyBinding guid="guidVSStd11" id="cmdidExecuteSelectionInInteractive" editor="guidCSharpEditorFactory" mod1="Control" key1="E" mod2="Control" key2="E" />
     <KeyBinding guid="guidInteractiveWindowCmdSet" id="cmdidCopyToInteractiveWindow" editor="GUID_TextEditorFactory" mod1="Control" key1="K" mod2="Control" key2="VK_RETURN" />
```
Comment (davkean) [inline src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct]: Now the question this leads to, with F# installed is this honored? They install a global command for this.

File: src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Query.cs [hunk 3]
```diff
@@ -42,12 +37,10 @@ public int QueryStatus(ref Guid pguidCmdGroup, uint commandCount, OLECMD[] prgCm
             {
                 return QueryVisualStudio97Status(ref pguidCmdGroup, commandCount, prgCmds, commandText);
             }
-#if INTERACTIVE
-            else if (pguidCmdGroup == InteractiveGuids.InteractiveCommandSetId)
+            else if (pguidCmdGroup == VSConstants.VsStd11)
             {
-                return QueryInteractiveCommandStatus(ref pguidCmdGroup, commandCount, prgCmds, commandText);
+                return QueryVisualStudio2011Status(ref pguidCmdGroup, commandCount, prgCmds, commandText);
```
Comment (davkean) [inline src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Query.cs]: There's no "VisualStudio2011", they mean VisualStudio11.

Comment (davkean) [inline src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Query.cs]: These are two different things.

File: src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct (hunk 2 above)
Comment (drognanar) [inline src/InteractiveWindow/VisualStudio/InteractiveWindow.vsct]: Yes, I checked and the keybindings for execute in interactive work for both C# and F#.

File: src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Query.cs (hunk 3 above)
Comment (drognanar) [inline src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Query.cs]: Made the change.

**Comment by drognanar** [comment]
test this please

**Comment by drognanar** [comment]
test this please

**Pull request merged**